docs/page-controller.md

```diff
@@ -130,8 +132,46 @@ function add_report_menu_item( $report_pages ) {
 add_filter( 'woocommerce_admin_report_menu_items', 'add_report_menu_item' );
 ```
 
+#### Example - Adding a New WooCommerce Admin Page
+
+Alternatively, register a regular page with the controller.
+
+```php
+wc_admin_register_page( array(
+	'id'       => 'my-example-page',
+	'title'    => __( 'My Example Page', 'my-textdomain' ),
+	'parent'   => 'woocommerce',
+	'path'     => '/example',
+	'nav_args' => array(
+		'order'  => 10,
+		'parent' => 'woocommerce',
+	),
+), );
+```
+
+Supply a React component on the client side. Be sure to use the same id supplied in PHP in `navArgs`.
+
+```javascript
+import { addFilter } from '@wordpress/hooks';
+
+const MyExamplePage = () => <h1>My Example Extension</h1>;
+
+addFilter( 'woocommerce_admin_pages_list', 'my-namespace', ( pages ) => {
+	pages.push( {
+		container: MyExamplePage,
+		path: '/example',
+		breadcrumbs: [ __( 'My Example Page', 'my-textdomain' ) ],
+		navArgs: {
+			id: 'my-example-page',
+		},
+	} );
+
+	return pages;
+} );
+```
+
 ### Further Reading
 
-* Check out the [`PageController`](../includes/page-controller/class-wc-admin-page-controller.php) class.
-* See how we're [connecting existing WooCommerce pages](../includes/page-controller/connect-existing-pages.php).
-* See how we're [registering Analytics Reports](../includes/features/analytics/class-wc-admin-analytics.php#L75).
+-   Check out the [`PageController`](../includes/page-controller/class-wc-admin-page-controller.php) class.
+-   See how we're [connecting existing WooCommerce pages](../includes/page-controller/connect-existing-pages.php).
+-   See how we're [registering Analytics Reports](../includes/features/analytics/class-wc-admin-analytics.php#L75).
```

Comment:
All of these links `404`'ed for me.

Reply:
Good call, thanks. https://github.com/woocommerce/woocommerce-admin/pull/5844/commits/4a557a852cce66b2ab0faea45279975eca09baa3